### PR TITLE
Mixin generation in addon

### LINKED ISSUE #4803: Generated tests for mixin use the wrong path when the recipe is used in addons
Steps to reproduce 

1. Create a new addon (ember addon ember-addon)
1. cd ember-addon
1. Generate a mixin `ember g mixin sample`
1. Run the tests `ember test`

Expected: 

* The default test should succeed (or be incomplete)

Actual: 

* The test fails with the message "Error: Could not find module `dummy/mixins/sample` imported from `dummy/tests/unit/mixins/sample-test". 

The import statement used is `import SampleMixin from '../../../mixins/sample';` When replaced with `import SampleMixin from 'ember-addon/mixins/sample';` it works. 

I could fix it by simply using modulePrefix on the import path, but I want to make sure I'm considering everything.

### PR DESCRIPTION
**Pull request by MiguelMadero**
Fixes #4803

### CODE REVIEW

File: blueprints/mixin-test/index.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ var testInfo = require('ember-cli-test-info');
 module.exports = {
   description: 'Generates a mixin unit test.',
   locals: function(options) {
-    return {
+  	return {
```
Comment (kellyselden) [inline blueprints/mixin-test/index.js]: indentation got screwy here.

File: blueprints/component-addon/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ var path               = require('path');
 var normalizeEntityName = require('ember-cli-normalize-entity-name');
 
 module.exports = {
-  description: 'Generates a component. Name must contain a hyphen.',
+  description: 'Generates an import wrapper.',
```
Comment (kellyselden) [inline blueprints/component-addon/index.js]: is this relevant?

File: blueprints/mixin-test/index.js (hunk 1 above)
Comment (MiguelMadero) [inline blueprints/mixin-test/index.js]: good catch. argh tabs vs spaces... I had the wrong setting on this new computer.

File: blueprints/component-addon/index.js (hunk 2 above)
Comment (MiguelMadero) [inline blueprints/component-addon/index.js]: Not relevant for this fix. I reverted it.

**Comment by rwjblue** [comment]
LGTM

**Comment by stefanpenner** [comment]
:+1:

**Comment by stefanpenner** [comment]
@homu r+

**Pull request merged**